stix_shifter_modules/nozomi/README.md

```diff
@@ -0,0 +1,535 @@
+        "query=alerts | where ip_src==\"1.1.1.1\" OR ip_dst==\"1.1.1.1\" | where record_created_at>=1704106800000 | where record_created_at<=1704844800000"
+    ]
+}
+```
+
+#### STIX Transmit results 
+
+```shell
+transmit nozomi "{\"host\":\"nozomi-xxxxxxxxx.vantage.nozominetworks.io\", \"port\":443}" "{\"auth\":{\"key_name\":\"XXXXXXX\", \"key_token\":\"xxxxxxxxxxxxxxxxxxxxxxxxxxxx\"}}" results "query=alerts | where ip_src==\"1.1.1.1\" OR ip_dst==\"1.1.1.1\" | where record_created_at>=1704106800000 | where record_created_at<=1704844800000"
+0
+1
+```
+
+#### STIX Transmit results - output
+```json
+{
+    "success": true,
+    "data": [
+        {
+            "id": "e2836408-55da-5d90-9601-398c1044ef16",
+            "time": 1704791849000,
+            "name": "Malformed traffic",
+            "type_name": "Malformed traffic",
+            "threat_name": "",
+            "counter": 1,
+            "description": "The UDP header in the packet contains a wrong data length. 1404 bytes are advertised, but 1380 bytes are found in the payload.",
+            "ack": false,
+            "note": null,
+            "risk": 7.0,
+            "id_src": "1.1.1.1",
+            "id_dst": "2.2.2.2",
+            "ip_src": "1.1.1.1",
+            "ip_src:info": null,
+            "ip_dst": "2.2.2.2",
+            "ip_dst:info": null,
+            "status": "open",
+            "mac_src": "00:d0:24:25:f9:54",
+            "mac_dst": "ff:ff:ff:ff:ff:ff",
+            "port_dst": null,
+            "port_src": null,
+            "protocol": "",
+            "transport_protocol": "unknown",
+            "severity": 10,
+            "zone_dst": "Broadcast",
+            "zone_src": "PlantSmartDevices",
+            "dst_roles": "other",
+            "src_roles": "other",
+            "label_dst": null,
+            "label_src": null,
+            "bpf_filter": "(ip host 1.1.1.1 and ip host 2.2.2.2) or (vlan and ip host 1.1.1.1 and ip host 2.2.2.2)",
+            "properties": {
+                "base_risk": 7.0,
+                "raised_by": "n2os_ids",
+                "is_dst_public": false,
+                "is_src_public": false,
+                "is_dst_node_learned": true,
+                "is_src_node_learned": true,
+                "mitre_attack_for_ics": {
+                    "source": {
+                        "levels": [
+                            "2"
+                        ]
+                    }
+                },
+                "is_dst_reputation_bad": false,
+                "is_src_reputation_bad": false,
+                "mitre_attack_enterprise": {
+                    "techniques": [
+                        {
+                            "id": "T1565",
+                            "name": "Data Manipulation",
+                            "tactic": "Impact"
+                        }
+                    ]
+                },
+                "incident_key_confidence:AnomalousPackets_10.80.68.90-255.255.255.255-": 1.0
```

Comment:
IP addresses are visible here (even if it's a range)

Reply:
Please make sure to check the other examples in here as well.

Reply:
Updated with obscured values